lib/posix/pthread_mutex.c

```diff
@@ -165,8 +168,7 @@ int pthread_mutex_trylock(pthread_mutex_t *m)
 int pthread_mutex_timedlock(pthread_mutex_t *m,
 			    const struct timespec *abstime)
 {
-	int32_t timeout = (int32_t)timespec_to_timeoutms(abstime);
-	return acquire_mutex(m, K_MSEC(timeout));
+	return acquire_mutex(m, K_MSEC((int32_t)timespec_to_timeoutms(abstime)));
```

Comment:
Feel free to ignore - but it looks like there is no functional changes here and we just make code much less readable.

Reply:
It saves a register 🤷🏼‍♂️

Reply:
Fixed!